documentation/src/main/asciidoc/search-dsl.asciidoc

```diff
@@ -1575,6 +1575,20 @@ include::{sourcedir}/org/hibernate/search/documentation/search/projection/Projec
 ----
 ====
 
+==== Elasticsearch: `took` and `explanation`
```

Comment:
This has nothing to do with projections, so it shouldn't be in the section of the documation relative to projections.
Maybe put this in the section about the query DSL, after the section `search-dsl-query-debugging-scores`?